requirements/requirements-ci.txt

```diff
@@ -76,7 +76,6 @@ jinja2==2.10.1
 jsonschema==3.0.1
 lazy-object-proxy==1.4.1
 lru-dict==1.1.6
-macholib==1.14            # via pyinstaller
```

Comment:
@ulope do we need this? I remember we had one thing that was removed unintentionally and was required for macos

Reply:
AFAIK we should leave this in for exactly that reason.